Documentation/Examples.md

```diff
@@ -71,6 +71,25 @@ label.textColor = R.color.appColors.textColor()
 
 There are some points to keep in mind when using Color palettes, see [About Colors](Colors.md)
 
+## Localized strings
+
+*Vanilla*
+```swift
+let welcomeMessage = NSLocalizedString("welcome.message", comment: "")
+let settingsTitle = NSLocalizedString("title", tableName: "Settings", comment: "")
+
+let welcomeName = String(format: NSLocalizedString("welcome.withName", comment: ""), locale: NSLocale.currentLocale(), "Alice")
```

Comment:
That constructor states that it formats
> according to the user’s default locale

I think that is the same as the current locale, but it's not a big thing. Let's keep it as is for now.